src/mame/skeleton/camirf1.cpp

```diff
@@ -0,0 +1,76 @@
+// license: BSD-3-Clause
+// copyright-holders:
+
+/*************************************************************
+
+Skeleton driver for Honeywell CAMIR-F1 wireless passive
+infrared motion sensor with built-in colour camera.
+
+Hardware based on a PIC18LF26K22 as main CPU with a Conexant
+CX93610 (DIFT JPEG Encoder with a 656 Camera Interface and
+Optional Microphone Input) and a IS25LQ040 Serial Flash.
+
+PCB:
+  _________    _    _________
+ /         |__| |__|         \
+|                             |
+|            _____        _   |
+|           |CAMERA------| |  |
+|           |SENSOR------| |  |
+|           |_____|------|_|  |
+|            _____     _____  |
+|           |CX93610  |PIC18LF26K22
+|           |     |   |    |  |
+|           |_____|   |    |  |
+|                     |____|  |
+|                  _____      |
+|                 |IS25LQ040  |
+|        Xtal                 |
+|       27 MHz                |
+|              ____           |
+|             /IR  |          |
+|            |SENSOR   __     |
+|   ___      |____/   |__|    |
+|  |___|                      |
+|           _________         |
+|__________|         |________|
+
+*************************************************************/
+
+#include "emu.h"
+
+namespace {
+
+
+class camirf1_state : public driver_device
+{
+public:
+	camirf1_state(const machine_config &mconfig, device_type type, const char *tag) :
+		driver_device(mconfig, type, tag)
+	{ }
+
+	void camirf1(machine_config &config) ATTR_COLD;
+};
+
+INPUT_PORTS_START( camirf1 )
+INPUT_PORTS_END
+
+void camirf1_state::camirf1(machine_config &config)
+{
+	// PIC18LF26K22
+}
```

Comment:
Is this core unique enough to have no CPU hooked up?

Reply:
Fixed, using PIC16 as the nearest choice.

Thx!